modules/google-pubsub/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "@erento/nestjs-module-google-pubsub",
   "version": "2.2.0",
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0"
```

Comment:
If you want to upgrade to node 16 and npm 8 here, any consumer of the new package also has to be upgraded to that versions. Do we want that? Or more specific question: Do we need node 16 or npm 8 for the package to run?